raiden/network/transport/matrix/transport.py

```diff
@@ -992,17 +991,16 @@ def _handle_sync_messages(self, sync_messages: MatrixSyncMessages) -> bool:
                 for text in room_messages:
                     all_messages.extend(self._handle_text(room, text))
 
-        # Remove this #3254
         for message in all_messages:
-            if isinstance(message, (Processed, SignedRetrieableMessage)) and message.sender:
```

Comment:
This was missed. I excluded `Processed` from the `Delivered` confirmation as the PR intended.